deploy/07_deploy_signer_bonds_escrow.ts

```diff
@@ -0,0 +1,16 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { deployer } = await getNamedAccounts()
+
+  await deployments.deploy("SignerBondsManualSwap", {
+    from: deployer,
+    log: true,
+  })
+}
+
+export default func
+
+func.tags = ["SignerBondsManualSwap"]
```

Comment:
Looks like in all other files the tags are the same as the second part of the file name that occurs after `<number>_deploy_`.

So maybe we could change the name of the file here?

`07_deploy_signer_bonds_escrow.ts` -> `07_deploy_signer_bonds_manual_swap.ts`

Reply:
I think it makes seanse. The `SignerBondsEscrow` is now called `SignerBondsManualSwap`- please see 874455749efe522f06e634c8e36ef977a4a3cb44.

Reply:
731808a1f3216994794a6b1ee0750f342bce7b07